src/App.js

```diff
@@ -0,0 +1,51 @@
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import Cookies from 'js-cookie';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import SplashPage from './components/SplashPage';
+import ProtectedRoute from './components/ProtectedRoute';
+import { loginAsync } from './redux/users/userSlice';
+import Doctors from './components/Doctors';
+import Layout from './components/Layout';
+import Details from './components/Details';
+import Appointments from './components/Appointments';
+import NewAppointment from './components/NewAppointment';
+import NewPasswordForm from './authentication/passwords/newPassword';
+import NewDoctor from './Pages/NewDoctor';
+import DeleteDoctor from './components/delete';
+
+const App = () => {
```

Comment:
- [ ] Please, add a little more styling to the desktop homepage, so they look more like the design.

Your Implementation | The Design
:---:|:---:
![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/2af8cd12-82c7-49fd-8dc8-f7f876e58815) | ![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/15582780-b949-495b-b88a-59163686da36)

That way, your project will look more professional. 👍